core/src/main/scala/org/apache/spark/scheduler/Pool.scala

```diff
@@ -21,6 +21,7 @@ import java.util.concurrent.{ConcurrentHashMap, ConcurrentLinkedQueue}
 
 import scala.collection.JavaConverters._
 import scala.collection.mutable.ArrayBuffer
+import scala.math.{max,min}
```

Comment:
(FYI, running `./dev/run-tests` will trigger style check first. Running the first part of this script before submitting more commits will show some comments I said here)